README.md

```diff
@@ -15,4 +15,6 @@ If you are running your API locally, you will need to configure vite to proxy an
 Install extension ESLint. Enter 'npm run lint' to get a error list in the terminal.
 To disable rule on next line comment '// eslint-disable-next-line' followed by the rule name e.g '// eslint-disable-next-line no-alert'.
 See .eslintrc.jc for rules. 
+To fix all lint errors 'npm run lint -- --fix'
```

Comment:
What are the changes to the files above this ReadMe it won't let me select them for some reason? - slnx, suo. read.lock, etc.

Reply:
+ 'To fix all lint errors 'npm run lint -- --fix''